WooCommerce/Classes/Authentication/Prologue/LoginProloguePages.swift

```diff
@@ -106,24 +106,32 @@ final class LoginProloguePageTypeViewController: UIViewController {
 
 private extension LoginProloguePageTypeViewController {
     func configureStackView() {
-        view.addSubview(stackView)
+        // Scroll view to contain all contents
+        let scrollView = UIScrollView()
+        view.addSubview(scrollView)
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: Constants.stackBottomMargin),
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
+        ])
+        scrollView.addSubview(stackView)
 
         // Stack view layout
         stackView.axis = .vertical
         stackView.alignment = .center
         stackView.spacing = Constants.stackSpacing
 
-        // Reduce centerYAnchor constraint priority to ensure the bottom margin has higher priority, so stack view is fully visible on shorter devices
-        let verticalCentering = stackView.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: Constants.stackVerticalOffset)
-        verticalCentering.priority = .required - 1
 
         // Set constraints
         stackView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            verticalCentering,
-            stackView.bottomAnchor.constraint(lessThanOrEqualTo: view.bottomAnchor, constant: Constants.stackBottomMargin),
-            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+            stackView.widthAnchor.constraint(equalTo: scrollView.widthAnchor),
+            stackView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            stackView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            stackView.topAnchor.constraint(equalTo: scrollView.topAnchor),
```

Comment:
nit: these could be
```suggestion
            scrollView.pinSubviewToAllEdges(stackView)
```

Reply:
Also fixed in c88ff1b90051f62fc80bcc00d099295d501a4a1d.